client-participation/js/strings/cy.js

```diff
@@ -0,0 +1,128 @@
+// Copyright (C) 2012-present, The Authors. This program is free software: you can redistribute it and/or  modify it under the terms of the GNU Affero General Public License, version 3, as published by the Free Software Foundation. This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero General Public License for more details. You should have received a copy of the GNU Affero General Public License along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+var s = {};
+
+s.agree = "Cytuno";
+s.disagree = "Anghytuno";
+s.pass = "Dim diddordeb / Ansicr";
+
+s.importantCheckbox = "Mae’r sylw yma’n bwysig";
```

Comment:
should prob remove all these fancy quotes, and markdown processor use smartquotes to do this on-the-fly